Source/WebInspectorUI/UserInterface/Workers/Formatter/JSFormatter.js

```diff
@@ -707,15 +711,19 @@ JSFormatter = class JSFormatter
         }
 
         if (nodeType === "Property") {
-            console.assert(tokenValue === ":" || tokenValue === "get" || tokenValue === "set" || tokenValue === "async" || tokenValue === "*" || tokenValue === "[" || tokenValue === "]" || tokenValue === "(" || tokenValue === ")", token);
+            console.assert(tokenValue === ":" || tokenValue === "static" || tokenValue === "get" || tokenValue === "set" || tokenValue === "async" || tokenValue === "*" || tokenValue === "[" || tokenValue === "]" || tokenValue === "(" || tokenValue === ")" || tokenValue === "#" || tokenValue === "=", token);
```

Comment:
NIT: maybe change this to a `[":", "static", ...].includes(tokenValue)` to make it less verbose?